LICENSE.md

```diff
@@ -3,7 +3,6 @@ Open Source Licenses
 
 The individual JUnit modules/artifacts use different open source licenses:
 
-- `junit-platform-surefire-provider` uses [Apache License v2.0](junit-platform-surefire-provider/LICENSE.md)
-- All other modules use [Eclipse Public License v2.0](junit-jupiter-api/LICENSE.md).
+- All modules use [Eclipse Public License v2.0](junit-jupiter-api/LICENSE.md).
```

Comment:
I think we should just replace this file with the EPL v2 now.

Reply:
✔️